app/master/build.py

```diff
@@ -278,6 +278,18 @@ def mark_failed(self, failure_reason):
         """
         self._state_machine.trigger(BuildEvent.FAIL, error_msg=failure_reason)
 
+    def mark_setup_failed(self, failure_reason):
+        """
+        Mark a build as failed and set a failure reason. Because setup failures don't have any logs, we put the build_id
+        in the setup_failed file for easier querying of worker logs.
+
```

Comment:
May I ask you to remove this line to keep with the consistency of the function comments in this file. FWIW, though, I also enjoy the space between params and overall functional comment, too, but consistency overrides preferences. :)